lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2348,8 +2348,8 @@ PropertyWrapperBackingPropertyInfoRequest::evaluate(Evaluator &evaluator,
   pbdPattern->setType(storageType);
   pbdPattern = TypedPattern::createImplicit(ctx, pbdPattern, storageType);
   auto pbd = PatternBindingDecl::createImplicit(
-      ctx, backingVar->getCorrectStaticSpelling(), pbdPattern,
-      /*init*/nullptr, dc, SourceLoc());
+      ctx, var->getCorrectStaticSpelling(), pbdPattern,
+      /*init*/ nullptr, dc, SourceLoc());
```

Comment:
Conventionally I think we *don't* have a space here, though I'm not sure why.

Reply:
Hmm clang-format did that, but can unrevert this if needed.

Reply:
Oh, well, clang-format is dictator. 😄